### DISCUSSION TITLE
miniDexed on the Raspberry Pi forum

### DISCUSSION THREAD

**Discussion by Gustavomurta** [probonopd/MiniDexed#523] [General]
Forgive me if the question is idiot, but has the minidexed project already been posted on the raspberry pi forum?

https://forums.raspberrypi.com/

**Comment by probonopd** [comment]
Wouldn't it be easier to search there instead of asking here?

**Comment by Gustavomurta** [comment]
Of course! I did the search and found nothing. If you don't want to disclose the project on the Raspberry forum, ok.
I think it can be an excellent channel for disseminating this excellent project.

**Reply by probonopd** [reply]
Hi @Gustavomurta would you like to make a post about MiniDexed there? That would be nice, because I don't have an account there. Thanks!

**Comment by Gustavomurta** [comment]
OK! thank you for your trust.
I've been developing tutorials on Raspberry PI (in Portuguese) since 2012. I'll promote miniDexed on the Raspberry Pi forum.

https://labdegaragem.com/forum/topics/conhecendo-o-raspberry-pi

https://labdegaragem.com/forum/topics/minidexed-incr-vel-sintetizador-com-rasperry-pi

**Comment by Gustavomurta** [comment]
I have now released the miniDexed project on the Raspberry pi forum.


**miniDexed - FM synthesizer!**
_https://forums.raspberrypi.com/viewtopic.php?t=354135_
